### ISSUE TITLE
Get Test Coverage Working Again

### ISSUE THREAD

**Issue by carrickr** [avalonmediasystem/avalon#1987]
### Description
Gives code coverage info on PRs. Currently turned off, has been broken since r6.

### Tasks
- [ ] Ensure travis is reporting to the Coveralls API
- [ ] Get it commenting on PRs
- [ ] Get the badges working

### Pointing
Probably a 2 or 3

**Comment by davidschober** [comment]
@phuongdh I heard you say you were working on this. If so, can you move it to in progress @mcwhitaker ^^^

**Comment by mcwhitaker** [comment]
Agreed.

**Comment by phuongdh** [comment]
Moved

**Comment by lealeelu** [comment]
Changes merged into develop. Waiting on Codeclimate to fix something on their end.
